capsules/object_speed/licenses/code.LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright 2020 Dilili Labs, Inc.
```

Comment:
Wrong year and aren't we using Aotu for copyrights?

Reply:
I'm going to continue using this, and then we can do a separate PR to change _every_ license in the repo.